src/main/java/org/junit/runners/model/RunnerBuilder.java

```diff
@@ -37,72 +37,73 @@
  * @since 4.5
  */
 public abstract class RunnerBuilder {
-    private final Set<Class<?>> parents = new HashSet<Class<?>>();
+  private final Set<Class<?>> parents = new HashSet<Class<?>>();
 
-    /**
-     * Override to calculate the correct runner for a test class at runtime.
-     *
-     * @param testClass class to be run
-     * @return a Runner
-     * @throws Throwable if a runner cannot be constructed
-     */
-    public abstract Runner runnerForClass(Class<?> testClass) throws Throwable;
+  /**
+   * Override to calculate the correct runner for a test class at runtime.
+   *
+   * @param testClass class to be run
+   * @return a Runner
+   * @throws Throwable if a runner cannot be constructed
+   */
+  public abstract Runner runnerForClass(Class<?> testClass) throws Throwable;
 
-    /**
-     * Always returns a runner, even if it is just one that prints an error instead of running tests.
-     *
-     * @param testClass class to be run
-     * @return a Runner
-     */
-    public Runner safeRunnerForClass(Class<?> testClass) {
-        try {
-            return runnerForClass(testClass);
-        } catch (Throwable e) {
-            return new ErrorReportingRunner(testClass, e);
-        }
+  /**
+   * Always returns a runner or null.
```

Comment:
@kcooney 
While debugging the code I saw `null` values.
This explains the null check in [AllDefaultPossibilitiesBuilder](https://github.com/junit-team/junit4/blob/24b8ee0bec2f2761b479bdd989275f19597955a3/src/main/java/org/junit/internal/builders/AllDefaultPossibilitiesBuilder.java#L25-L30)